client/lib/analytics/utils/product-to-ga-item.ts

```diff
@@ -10,12 +10,17 @@ export type GaItem = {
 	price: number;
 };
 
-export function jetpackProductToGaItem( product: ResponseCartProduct, currency: string ): GaItem {
+export function productToGaItem(
+	product: ResponseCartProduct,
+	currency: string,
+	sendToJetpack = false
+): GaItem {
+	const item_brand = sendToJetpack ? GA_PRODUCT_BRAND_JETPACK : GA_PRODUCT_BRAND_WPCOM;
```

Comment:
Cannot we distinguish the brand somehow from the `ResponseCartProduct` rather than passing the artificial flag `sendToJetpack`?

Reply:
I agree, passing this artificial property down through the functions isn't the cleanest way.

Reply:
Thanks! I've made the brand here independent of the tracking environment. I realized that these shouldn't be coupled together, as we'er currently also firing Jetpack product orders on our Enhanced Ecommerce tracking (Universal Analytics). 

We're now parsing the brand from the cart items. And the tracking environment is only for one thing (which Gtag to send the event to), with no other side effects on the logic.